.github/workflows/integration-install.yml

```diff
@@ -11,6 +11,30 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
+      - name: Get Draft cli version
```

Comment:
It might be better to use action script rather than writing your own code here. It's an officially supported way of doing this with built-in retries

https://github.com/actions/github-script

Reply:
I think I will end up again passing GITHUB_TOKEN to the JavaScript modules in the env variables, if I use this action. Is there any issue with having the script like this ? Please let me know if my understanding is wrong. Thanks for the suggestion.

Reply:
This action is safe. GITHUB_TOKEN is safe. You don't have to specify any token inputs when using actions script it will automatically use the GITHUB_TOKEN. There is no issue with using this script and it's the recommended (and official) approach.

Reply:
Implemented using action-script and runs are successful now - https://github.com/Azure/draft/actions/runs/5447836986/jobs/9910315437